cmd/jaeger/integration/fixtures/grpc_config.yaml

```
@@ -0,0 +1,23 @@
+service:
+  extensions: [jaeger_storage]
+  pipelines:
+    traces:
+      receivers: [otlp]
+      exporters: [jaeger_storage_exporter]
+
+extensions:
+  jaeger_storage:
+    grpc:
+      external-storage:
```

```suggestion
      some-external-storage:
```
With the current name the config reads as if this is a part of the actual config structure; adding "some-" emphasizes that it's just a user-given name